code/datums/abilities/flock/flockmind.dm

```diff
@@ -92,6 +92,10 @@
 
 	var/turf/T = get_turf(F)
 
+	if (F)
+		if (tgui_alert(F,"Would you like to spawn a rift?","Spawn Rift?",list("Yes","No")) != "Yes")
+			return TRUE
+
```

Comment:
I'd move this below the area checks so you can use it to see if the area you've selected is valid without actually deploying.